SSA/Projects/InstCombine/Tactic.lean

```diff
@@ -46,6 +47,8 @@ macro "simp_alive_meta" : tactic =>
       dsimp (config := {failIfUnchanged := false }) only [ConcreteOrMVar.instantiate]
       dsimp (config := {failIfUnchanged := false }) only [InstcombineTransformDialect.instantiateMTy]
       dsimp (config := {failIfUnchanged := false }) only [ConcreteOrMVar.instantiate_mvar_zero'']
+      -- How can I avoid the `simp! only`and use a plain `simp only`?
+      simp! (config := {failIfUnchanged := false }) only [ConcreteOrMVar.instantiate_list]
```

Comment:
Can I avoid here the `simp!`?

Reply:
@bollu and I debugged for an hour and could not find the solution. We opened an issue to be able to make progress: https://github.com/opencompl/ssa/issues/349